scripts/zones/Riverne-Site_A01/IDs.lua

```diff
@@ -18,8 +18,11 @@ zones[xi.zone.RIVERNE_SITE_A01] =
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
         CONQUEST_BASE                 = 7064, -- Tallying conquest results...
         A_GLOWING_MIST                = 7259, -- A glowing mist of ever-changing proportions floats before you...
+        MEMBERS_OF_YOUR_PARTY         = 7535, -- Currently, # members of your alliance (including yourself) have clearance to enter the battlefield.
```

Comment:
You're going to want to check for additional missing Text IDs that should be defined for battlefields.  Easiest way to know what is absolutely required is by searching `.text.` in battlefield.lua

Reply:
![image](https://github.com/LandSandBoat/server/assets/65316697/7db267c0-dfa2-434a-a541-82d5ef2f2cbf)